code/modules/power/supermatter/supermatter.dm

```diff
@@ -274,6 +286,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	)
 	AddElement(/datum/element/connect_loc, loc_connections)	//Speficially for the tram, hacky
 
+	AddElement(/datum/element/supermatter_crystal, CALLBACK(src, .proc/wrench_act_callback), CALLBACK(src, .proc/consume_callback))
```

Comment:
Callbacks don't really work in elements. You'd either need to 

A. Directly pass the proc path into the element and use `call()`
B. Make this an element